input/examples/patient-example-sex-and-gender.xml

```diff
@@ -26,15 +30,7 @@
                 </coding>
             </valueCodeableConcept>
         </extension>
-        <extension url="period">
-            <valuePeriod>
```

Comment:
Why is this period removed? No reason for removal in the PR.

Reply:
Reinstated accidentally removed code